dal/src/main/java/com/craftsmanship/tfm/dal/grpc/server/GrpcServer.java

```diff
@@ -86,7 +86,7 @@ public ItemPersistenceServiceImpl(DataAccess dataAccess) {
     public void create(CreateItemRequest request, io.grpc.stub.StreamObserver<CreateItemResponse> responseObserver) {
       logger.info("CREATE RPC CALLED");
       GrpcItem grpcItem = request.getItem();
-      Item item = new Item(grpcItem.getName());
+      Item item = new Item(grpcItem.getName(), grpcItem.getPrice(), grpcItem.getQuantity());
```

Comment:
why not use getItemFromGrpcItem ?

Reply:
I agree with the comments.
I create a new commit with modifications.